### PR TITLE
Script for DKIM Reg

### PR DESCRIPTION
**Pull request by saleel**
- Add scripts package. 
- Modify dkim key script to work with new DKIMRegistry.sol (Use poseidon_hash(pubkey))
- Remove defaults from DKIMregistry.sol
- Add option for setting multiple keys for a domain in one tx in DKIMregistry.sol
- Remove MailServer.sol

### CODE REVIEW

File: packages/scripts/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@zk-email/scripts",
```
Comment (Divide-By-0) [inline packages/scripts/package.json:2]: this should just go in helpers imo?

Comment (Divide-By-0) [reply packages/scripts/package.json:2]: or this dkim selector archive site can be its own site

Comment (saleel) [reply packages/scripts/package.json:2]: > this should just go in helpers imo?

I thought of it, but helpers is a sdk that other people would be importing to their code. We don't want these scripts to be part of it right.

**Comment by Divide-By-0** [comment]
Typescript errors for me:
Cannot find module '@zk-email/contracts/out/DKIMRegistry.sol/DKIMRegistry.json' or its corresponding type declarations.ts(2307)
import { abi } from "@zk-email/contracts/out/DKIMRegistry.sol/DKIMRegistry.json";

And I also don't see `/out` in contracts...

**Comment by saleel** [comment]
> Typescript errors for me: Cannot find module '@zk-email/contracts/out/DKIMRegistry.sol/DKIMRegistry.json' or its corresponding type declarations.ts(2307) import { abi } from "@zk-email/contracts/out/DKIMRegistry.sol/DKIMRegistry.json";
> 
> And I also don't see `/out` in contracts...

They will be generated when you run `yarn build` in contracts.

**Pull request merged**